tools/helpers.py

```diff
@@ -16,3 +16,10 @@ def get_remote_data(url):
     req.add_header('Accept', 'application/vnd.github.v3+json')
     json_response = urlopen(req, timeout=240).read()
     return json_response
+
+
+def get_weekly_data(url):
+    """Form Request object from URL and fetch weekly data."""
+    req = Request(url)
+    json_response = urlopen(req, timeout=240).read()
+    return json_response
```

Comment:
why not reusing `get_remote_data()` they do the same thing.

Reply:
That's a good question, @karlcow...

I think the `Accept` header for github media confused me. I wasn't sure if it applied in this case. If it's not a request directly to the github API, why aren't we using plain `application/json` as the accepted type?

Reply:
@laghee Probably a copy paste mistake from me and/or something I was planning initially to do through GitHub. I don't remember now. But you can change it indeed to just `application/json`. 

Thanks.

Reply:
Ah, OK! I'll do that and consolidate then. Thanks!

Reply:
@karlcow Done. Safe travels!